Actions/DetermineDeploymentEnvironments/DetermineDeploymentEnvironments.ps1

```diff
@@ -144,6 +144,7 @@ else {
             "Branches" = @()
             "BranchesFromPolicy" = @()
             "Projects" = '*'
+            "DependencyInstallMode" = "install"  # ignore, install or upgrade
```

Comment:
Shouldn't this (and the other settings) just be part of https://github.com/microsoft/AL-Go/blob/e5f2df3b796102c8ef4b018e797550e974b5554a/Actions/AL-Go-Helper.ps1#L552?

Reply:
Problem here is that `DeployTo<environmentName>` cannot be pre-set in settings as we don't know the environment name.

This maybe could have been made differently (and more json strict) by using

```json
DeployTo: [
    {
        "environmentName": "environmentName",
        "settings": {
            "settings": xxx
        }
    }
]
```

And maybe we should shift to do something like that - but that will require us to have a mechanism to upgrade settings as part of upgrade AL-Go System Files.

That is not for this PR.